pkg/controllers/workspace_controller.go

```diff
@@ -388,7 +393,7 @@ func (c *WorkspaceReconciler) applyAnnotations(ctx context.Context, wObj *kdmv1a
 	}
 
 	//TODO generate more strong random service name
-	serviceObj := k8sresources.GenerateLoadBalancerService(ctx, fmt.Sprint("workspace", "-scv", rand.Intn(100)), wObj.Namespace, serviceType, wObj.Resource.LabelSelector.MatchLabels)
+	serviceObj := k8sresources.GenerateLoadBalancerService(ctx, fmt.Sprint("workspace", "-scv", rand.Intn(100_000)), wObj.Namespace, serviceType, wObj.Resource.LabelSelector.MatchLabels)
```

Comment:
This was increased to prevent duplicate service name? maybe we use date or hash instead

Reply:
This is only temp but we need to design a proper way for all generated names (deployments, services, machines)